server/Server.hpp

```diff
@@ -28,10 +30,20 @@
 class Server {
 private:
 
+	std::vector<pollfd>::iterator it;
+	std::map<int, connection>::iterator mt;
+	std::map<int, Response>::iterator resps;
```

Comment:
we should also rename `Response` to `response` maybe to have matching cases

Reply:
we can refactor that later.